test/e2e/logger/test_raw_logger.py

```diff
@@ -30,14 +32,67 @@
 
 
 kserve_client = KServeClient(config_file=os.environ.get("KUBECONFIG", "~/.kube/config"))
+annotations = {"serving.kserve.io/deploymentMode": "RawDeployment"}
 
 
 @pytest.mark.raw
 @pytest.mark.asyncio(scope="session")
 async def test_kserve_logger(rest_v1_client):
     msg_dumper = "message-dumper-raw"
-    annotations = {"serving.kserve.io/deploymentMode": "RawDeployment"}
+    before(msg_dumper)
 
+    service_name = "isvc-logger-raw"
+    predictor = V1beta1PredictorSpec(
+        min_replicas=1,
+        logger=V1beta1LoggerSpec(
+            mode="all",
+            url="http://"
+            + msg_dumper
+            + "-predictor"
+            + "."
+            + KSERVE_TEST_NAMESPACE
+            + ".svc.cluster.local",
+        ),
+        sklearn=V1beta1SKLearnSpec(
+            storage_uri="gs://kfserving-examples/models/sklearn/1.0/model",
+            resources=V1ResourceRequirements(
+                requests={"cpu": "10m", "memory": "128Mi"},
+                limits={"cpu": "100m", "memory": "256Mi"},
+            ),
+        ),
+    )
+    base_test(msg_dumper, service_name, predictor, rest_v1_client)
+
+
+@pytest.mark.rawcipn
+async def test_kserve_logger_cipn(rest_v1_client):
```

Comment:
IMHO, [test_raw_deployment.py](https://github.com/kserve/kserve/blob/master/test/e2e/predictor/test_raw_deployment.py) would had been a better location for the new test.

Reply:
I don't remember exactly the reason that led me to put this test here, I guess it was something related to the logs messages to make sure those are also added for clusterIP.